timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/server/endpoints/v2/Gremlin.java

```diff
@@ -278,11 +278,18 @@ private String getVertexType(Vertex vertex) throws IOException {
   }
 
   private void dumpVertex(Vertex vertex, StringBuilder result) {
+    ObjectMapper map = new ObjectMapper();
```

Comment:
Objectmapper kan een field worden en worden hergebruikt in zowel dumpVertex als dumpEdge.